src/components/dealer-header/dealer-header.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  Component, Element, Prop, State, Watch,
+} from '@stencil/core';
+
+@Component({
+  tag: 'c-dealer-header',
+  styleUrl: 'dealer-header.scss',
+  shadow: true,
+})
+export class HelloWorld {
+  /** Dealer name */
+  @Prop() dealerName: string;
```

Comment:
Might want to use what regular header uses for site-name instead of dealer-name?

Same with site-name and dealer-site